js/components/portal-dashboard/answers/answer-modal.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { PureComponent } from "react";
+import { Modal } from "react-bootstrap";
+import SmallCloseIcon from "../../../../img/svg-icons/small-close-icon.svg";
+import { QuestionTypes } from "../../../util/question-utils";
+import { renderHTML } from "../../../util/render-html";
+
+import css from "../../../../css/portal-dashboard/answers/answer-modal.less";
+import "../../../../css/portal-dashboard/answers/answer-modal.css";
```

Comment:
I would add a comment about why this css import is needed.

Reply:
Something about how the CSS module import mutates the class names so that they are dynamic, but we needed to make a change to a static class name (`modal-content`) that exists in the imported modal dialog library.  The CSS import does that.  It lets us specify style on a statically named CSS class.